backends/openvino/runtime/OpenvinoBackend.hpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) Intel Corporation
+ *
+ * Licensed under the BSD License (the "License"); you may not use this file
+ * except in compliance with the License. See the license file in the root
+ * directory of this source tree for more details.
+ */
+
+#ifndef OPENVINO_BACKEND_HPP
+#define OPENVINO_BACKEND_HPP
+
+#include <memory>
+#include <iostream>
+#include <openvino/openvino.hpp>
+
+#include <executorch/runtime/backend/interface.h>
+#include <executorch/runtime/core/error.h>
+#include <executorch/runtime/core/evalue.h>
+#include <executorch/runtime/core/exec_aten/util/dim_order_util.h>
+#include <executorch/runtime/core/exec_aten/util/scalar_type_util.h>
+
+using namespace std;
+using executorch::aten::ScalarType;
+using executorch::runtime::ArrayRef;
+using executorch::runtime::Backend;
+using executorch::runtime::BackendExecutionContext;
+using executorch::runtime::BackendInitContext;
+using executorch::runtime::CompileSpec;
+using executorch::runtime::DelegateHandle;
+using executorch::runtime::Error;
+using executorch::runtime::EValue;
+using executorch::runtime::FreeableBuffer;
+using executorch::runtime::MemoryAllocator;
+using executorch::runtime::Result;
```

Comment:
Typically I would prefer to alias namespace if you want to shorten typing but blanket using entire namespace reduces readability

Reply:
Removed blanket namespaces:
https://github.com/ynimmaga/executorch/blob/0efa875b88def668a71d9357f4d75079e06f1e3c/backends/openvino/runtime/OpenvinoBackend.h#L21